### DISCUSSION TITLE
Show character and word count in quote submission textarea

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1996] [Ideas] [answered]
It would be helpful if the quote submission modal showed how many characters the quote entered has so a user can know if they have passed the 60 character minimum, also maybe adding an notification working the user that their quote is too short if their quote is less than 60 characters. Some possible locations for this would be here:
![Screenshot 2021-10-11 9 55 32 PM](https://user-images.githubusercontent.com/79121360/136894055-28ce1d72-8ac3-4fab-a26a-159a531550a8.png)

**Comment by tajhans** [comment]
Maybe a drop down menu below source?

**Comment by Miodec** [comment, marked as answer]
Sure, ill add it.
